softserve-orm/src/main/java/com/softserve/task4/orm/client/MySQLConnector.java

```diff
@@ -0,0 +1,54 @@
+package com.softserve.task4.orm.client;
+
+import com.softserve.task4.orm.processes.CreateTable;
+import com.softserve.task4.orm.processes.InsertColumns;
+import com.softserve.task4.orm.processes.UpdateColumns;
+
+import java.io.FileInputStream;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+
+/**
+ * Created by ${JDEEK} on ${11.11.2018}.
+ */
+public class MySQLConnector {
+
+    private Connection connection;
+    private Statement statement;
+
+    public MySQLConnector() {
+    }
+
+    public Connection connect() throws Exception {
+        Properties properties = new Properties();
+        FileInputStream stream = new FileInputStream("C:\\Gradle\\gradle-4.8.1\\softserve-orm\\src\\main\\resources\\database.properties");
```

Comment:
Please don`t use an absolute path.
Your project will not work on my laptop, for example.